src/book/03-guides/04-general/04-visualmesh.mdx

```diff
@@ -51,41 +87,40 @@ These lines tell us that the working directory of the container will be the `/wo
 docker build . --pull -t visualmesh:latest
 ```
 
+<details>
+
+<Summary>What does this command do?</Summary>
```

Comment:
In the sections with the bullet-points breaking down the command I've also merged these into detail tags to save screen space and make it easier to see